src/core/evaluator.js

```diff
@@ -1640,19 +1650,22 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
                 xObjStateManager.transform(matrix);
               }
 
+              // enqueue textContent chunk before call for sub-form.
+              enqueueChunk();
               next(self.getTextContent({
                 stream: xobj,
                 task,
                 resources: xobj.dict.get('Resources') || resources,
                 stateManager: xObjStateManager,
                 normalizeWhitespace,
                 combineTextItems,
+                sink
               }).then(function (formTextContent) {
-                Util.appendToArray(textContent.items, formTextContent.items);
-                Util.extendObj(textContent.styles, formTextContent.styles);
+                // Util.appendToArray(textContent.items, formTextContent.items);
+                // Util.extendObj(textContent.styles, formTextContent.styles);
 
-                xobjsCache.key = name;
-                xobjsCache.texts = formTextContent;
+                // xobjsCache.key = name;
+                // xobjsCache.texts = formTextContent;
```

Comment:
>  All we need to do is wrap sink and pass wrapper for inner processing: wrapper will redirect data to the original sink and store passed items.

So, wrapper for sink maybe something like: `let sinkWrapper = { sink: sink, items = [] }` ?
And instead of passing `sink` to `getTextContent()`, we can pass `sinkWrapper`.